extension/src/paymentHandler/klarna-make-payment.handler.js

```diff
@@ -1,10 +1,11 @@
+const _ = require('lodash')
 const ctpClientBuilder = require('../ctp')
 const makePaymentHandler = require('./make-payment.handler')
 const config = require('../config/config')
 
 const ADYEN_PERCENTAGE_MINOR_UNIT = 10000
-const DEFAULT_PAYMENT_LANGUAGE = 'en'
```

Comment:
it seems that the doc of _getLocales function in this file is not up to date. see:
```...There will always be a locale `DEFAULT_PAYMENT_LANGUAGE` as a default fallback. ```
similar for ```If it's not set, it will pick `DEFAULT_PAYMENT_LANGUAGE`.```

Reply:
I removed the comments as they're not very useful: https://github.com/commercetools/commercetools-adyen-integration/pull/733/commits/683941ed15303d093814c97e98d26ef91b7a91d6